cwms-data-api/src/main/java/cwms/cda/data/dto/watersupply/WaterUserContract.java

```diff
@@ -0,0 +1,241 @@
+/*
+ *
+ * MIT License
+ *
+ * Copyright (c) 2024 Hydrologic Engineering Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package cwms.cda.data.dto.watersupply;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import cwms.cda.data.dto.CwmsDTO;
+import cwms.cda.data.dto.CwmsId;
+import cwms.cda.data.dto.LookupType;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV1;
+import java.util.Date;
+
+@FormattableWith(contentType = Formats.JSONV1, formatter = JsonV1.class)
+@JsonDeserialize(builder = WaterUserContract.Builder.class)
+public class WaterUserContract extends CwmsDTO {
+
+    @JsonProperty(required = true)
+    private final WaterUser waterUser;
+    @JsonProperty(required = true)
+    private final CwmsId contractId;
+    @JsonProperty(required = true)
+    private final LookupType waterContract;
```

Comment:
this variable name is ambiguous and results in confusing json.
  "contract-id": {
    "office-id": "MVR",
    "name": "TEST_CONTRACT"
  },
  "water-contract": {
    "office-id": "MVR",
    "display-value": "Test Display Value",
    "tooltip": "Test Tooltip",
    "active": true
  },

Reply:
Will rename waterContract to contractType